src/rest-server/deploy/rest-server.yaml.template

```diff
@@ -106,11 +106,12 @@ spec:
         - name: GITHUB_PATH
           value: {{ cluster_cfg['rest-server']['github-path'] }}
 {% endif %}
-{% if cluster_cfg['layout']['kubernetes']['api-servers-url'].startswith('https://') %}
-        - name: K8S_APISERVER_CA_FILE
```

Comment:
`K8S_APISERVER_CA_FILE` is still used in the code, do you need to delete it here?

Reply:
Yes, the env is kept in non-rbac mode for expandability in the future. And in rbac enabled cluster, thoses two values are always a hardcode value, so just remove them from deployment code at this stage.